### PR TITLE
Fixing GitHub issue #29538

### PR DESCRIPTION
**Pull request by bgaydosrh**
**4.7 merge conflict resolved with 
https://github.com/openshift/openshift-docs/pull/32630

4.6 merge conflict resolved with 
https://github.com/openshift/openshift-docs/pull/32643**

This is in response to https://github.com/openshift/openshift-docs/issues/29538 from @peterclauterbach.

Use Netify link for Test Build.

Upon approval will merge and cp to enterprise-4.5 enterprise-4.6, enterprise-4.7, and enterprise-4-8

cc: @ctomasko

### CODE REVIEW

**Comment by bgaydosrh** [comment]
@peterclauterbach can you review and approve this change? Thanks, Bob

**Review by peterclauterbach — APPROVED** [review]
LGTM

File: virt/install/preparing-cluster-for-virt.adoc [hunk 1]
```diff
@@ -31,6 +31,8 @@ In {VirtProductName} clusters installed using installer-provisioned infrastructu
 Without an external monitoring system or a qualified human monitoring node health, virtual machines lose high availability.
 ====
 
+* Shared storage is required to enable Live Migration.
```
Comment (sjhala-ccs) [inline virt/install/preparing-cluster-for-virt.adoc:34]: s/Live Migration/live migration

**Review by sjhala-ccs — COMMENTED** [review]
Just one nit, otherwise LGTM

File: virt/install/preparing-cluster-for-virt.adoc (hunk 1 above)
Comment (bgaydosrh) [reply virt/install/preparing-cluster-for-virt.adoc]: Done, thanks @sjhala-ccs

**Pull request merged**